frontend/public/components/events.jsx

```diff
@@ -480,26 +501,27 @@ class EventStream extends React.Component {
     if (error) {
       statusBtnTxt = (
         <span className="co-sysevent-stream__connection-error">
-          Error connecting to event stream{_.isString(error) && `: ${error}`}
+          {t('events~Error connecting to event stream')}
+          {_.isString(error) && t('events~: {{error}}', { error })}
         </span>
       );
       sysEventStatus = <ErrorLoadingEvents />;
     } else if (loading) {
-      statusBtnTxt = <span>Loading events...</span>;
+      statusBtnTxt = <span>{t('events~Loading events...')}</span>;
       sysEventStatus = <Loading />;
     } else if (active) {
-      statusBtnTxt = <span>Streaming events...</span>;
+      statusBtnTxt = <span>{t('events~Streaming events...')}</span>;
     } else {
-      statusBtnTxt = <span>Event stream is paused.</span>;
+      statusBtnTxt = <span>{t('events~Event stream is paused.')}</span>;
     }
 
     const klass = classNames('co-sysevent-stream__timeline', {
       'co-sysevent-stream__timeline--empty': !allCount || !count,
     });
     const messageCount =
       count < maxMessages
-        ? `Showing ${pluralize(count, 'event')}`
-        : `Showing ${count} of ${allCount}+ events`;
+        ? t('events~Showing {{count}} event', { count })
+        : t('events~Showing {{count}} of {{allCount}}+ events', { count, allCount });
```

Comment:
I think using `{{count}}` in this way won't pluralize correctly. `allCount` needs to be `count` since it's what determines the plural for "events."

Reply:
@spadgett, ok, i don't want to use pluralization here...events should always be plural. I can use another variable name aside from count.